src/components/NavBar.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+const GENRES = ['ALL', 'DOCUMENTARY', 'COMEDY', 'HORROR', 'CRIME'];
+
+class NavBar extends React.PureComponent {
```

Comment:
Please use either this NavBar component or the one defined in src/App.js

Reply:
Fixed.